generator.yaml

```diff
@@ -188,6 +188,9 @@ resources:
       OutputDataConfig.KMSKeyID:
         late_initialize:
           min_backoff_seconds: 5
+      ResourceConfig.InstanceCount:
```

Comment:
is this change in behaviour?

what happens if user is using instance groups only?

Reply:
Added retry strategy to integration tests and added unit tests.

>is this change in behavior?
Yes, instance count is optional because the user can specify Instance group. However the server side default of instanceCount is 0.
> what happens if user is using instance groups only?
Instance count will be late initialized to 0.

Reply:
this is a very strange behaviour